examples/joint_inference/helmet_detection_inference/little_model/little_model.py

```diff
@@ -159,9 +154,17 @@ def main():
         img_rgb = cv2.cvtColor(input_yuv, cv2.COLOR_BGR2RGB)
         nframe += 1
         LOG.info(f"camera is open, current frame index is {nframe}")
-        inference_result = InferenceResult(
-            *inference_instance.inference(img_rgb))
-        output_deal(inference_result, nframe, img_rgb)
+        is_hard_example, final_result, edge_result, cloud_result = (
```

Comment:
From the user view, he may only care about the final inference result.

Reply:
We had a discussion before， user may need to compare the difference in the inference results of the big/little model. In this example, the user keeps all hard samples at the same time, and is considered to be able to find out any samples with low-confidence, for post-processing

Reply:
You also need to consider the users that don't care the image is hard or not.